cypress/tsconfig.json

```diff
@@ -1,17 +1,10 @@
 {
+  "extends": "../tsconfig.json",
```

Comment:
Inherit the options from the parent `tsconfig`, which already sets up these values and ensure that these files are in sync except for the types and includes.